src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,74 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
+  const [searchParam] = useSearchParams();
+  const sex = searchParam.get('sex') || '';
+  const nameSearch = searchParam.get('name') || '';
+  const centuries = searchParam.getAll('centuries') || [];
+  const sort = searchParam.get('sort') || '';
```

Comment:
The `sort` parameter is retrieved but not used in the component. Ensure that this parameter is integrated into your sorting logic for the `PeopleTable`.